### ISSUE TITLE
Question - No new issue. :)

### ISSUE THREAD

**Issue by ghost** [Galts-Gulch/avarice#33]
How are you with machine learning? Would it be possible to set up a machine to learn when to make trades? The reason I ask is that I've got an Azure account and I see Machine learning is part of my subscription. So just thought I'd see what your thoughts were regarding that.

**Comment by RealJohnGalt** [comment]
Apologies for the late response yet again -- really need to fix email notifications.

Some forms of machine learning can definitely be profitable. I have a support vector machine (SVM) based C++ bot I've been working on for awhile that has been long-term profitable for OKCoin spot trading. I have a bit of experience with GA for decision making, and have found that can be profitable as well. Most recently, I've been working on trying to get a basic RNN system profitable (and trying to get it functional to begin with...).

The biggest time drain when it comes to any form of ML for trading is testing inputs...and more testing, testing testing (from what I've found).

Beyond this minor experience, I can't say much else on it. If you have the time, it's definitely worth the experimenting. There are quite a few great libraries and PDFs out there that can get you started.

Regarding Azure, I really can't comment. I'm planning on looking into it more since you've piqued my interest.

**Comment by ghost** [comment]
Yeah give Azure a look, it's been getting a lot of positive feedback from what I can tell. So would love to setup Machine learning, or even a neural network to assist with the trading methodology.

**Comment by ghost** [comment]
Check this out if you have some time. I've been experimenting with predictive analytics.
http://cloudacademy.com/blog/azure-machine-learning/
